### ISSUE TITLE
Transcription model producing empty output

### ISSUE THREAD

**Issue by mabarber92** [mittagessen/kraken#523]
As noted in today's meeting the model trained from scratch on Arabic manuscripts fails to recognise any text. I have tried running it on binarized and un-binarized. The output in eScriptorium is like this:

![image](https://github.com/mittagessen/kraken/assets/46000359/2c3e9e37-1fe1-4afa-b70b-e81df353b0e6)

The model being used is this:
[ms_scratch.zip](https://github.com/mittagessen/kraken/files/11883408/ms_scratch.zip)

I believe the model may have an error, as it doesn't have an accuracy rate. However, eScriptorium does not error (and I'm assuming that Kraken isn't erroring in the backend either, as it produces output).

**Comment by mabarber92** [comment]
@mittagessen @dasmiq Would it make sense to figure this out within this issue?

**Comment by dasmiq** [comment]
This was the page I was trying to train on: https://github.com/OpenITI/arabic_ms_data/blob/main/firuzabadi_al_qamus_al_muhit/sbzb_glaser_33/sbzb_glaser_33.pdf_page_11.xml
(I've seen the same issue of zero accuracy with larger training runs, but hopefully this is sufficient to test.)

**Comment by dasmiq** [comment]
@mabarber92 To correct what you said, my second training run was on the same manuscript page, but I initialized the model from a print model instead of starting from scratch.

**Comment by colibrisson** [comment]
> David pulled the latest version of Kraken from the main branch but does not get the training loss. Can you suggest the correct branch to use?

You can monitor the training using Tensorboard. You just need to add the `--logger tensorboard` and `--log-dir ./` arguments to your command.

**Comment by dasmiq** [comment]
Yes, I've used smaller learning rates in many experiments. But I sent you the results using the default parameters for simplicity. Just tell me what parameters you'd like to see to debug this issue.

**Comment by dstoekl** [comment]
I usually set 
-B 1 
-r 0.0001
-w 0
for greater batchsize increase -r by squareroot of -B

a single page is not enough GT to converge however.

**Comment by dasmiq** [comment]
Thank you @dstoekl I know a single page is small, but this issue with training from scratch persists at larger amounts of training data. Could you suggest an amount of data and training parameters you would like us to run?

**Comment by colibrisson** [comment]
You should first use Tensorboard to check if your training loss is decreasing.

**Comment by mittagessen** [comment]
@dasmiq The training loss probably gets overwritten by the pytorch-lightning progress bar when the validation loss is available. Tensorboard logging is probably the quickest way to check it though.
